invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js

```diff
@@ -614,6 +621,16 @@ export class RDMDepositForm extends Component {
                           <Grid.Column width={16} className="pt-10">
                             <PublishButton fluid />
                           </Grid.Column>
+
+                          <Grid.Column width={16} className="pt-0">
+                            {(record.is_draft === null || permissions.can_manage) && (
```

Comment:
Can it be `null`? Shouldn't it be true/false?

Reply:
`is_draft` is null after just clicking on the "New upload" and rendering an empty deposit form